src/EPR.Accreditation.API/Profiles/AccreditationProfile.cs

```diff
@@ -64,6 +64,11 @@ public AccreditationProfile()
                 .ReverseMap()
                 .MapOnlyNonDefault();
 
+            CreateMap<Data.SaveAndContinue, DTO.SaveAndContinue>()
+                .MapOnlyNonDefault()
+                .ReverseMap()
+                .MapOnlyNonDefault();
```

Comment:
We won't need "MapOnlyNonDefault" for save and continue

Reply:
Removed